include/libtorrent/peer_connection.hpp

```diff
@@ -458,9 +458,9 @@ namespace libtorrent
 
 		stat const& statistics() const { return m_statistics; }
 		void add_stat(boost::int64_t downloaded, boost::int64_t uploaded);
-		void sent_bytes(int bytes_payload, int bytes_protocol);
-		void received_bytes(int bytes_payload, int bytes_protocol);
-		void trancieve_ip_packet(int bytes, bool ipv6);
```

Comment:
I'm having second thoughts about turning all these into unsigned. it feels scary